### PR TITLE
Export options and formats

### PR DESCRIPTION
**Pull request by DanielHauschildt**
Added support `foreground`, `background` and `mask` export type
Added support for `webp` and `jpeg` export formats.
Experimental: Added support for preloading all assets.

### CODE REVIEW

File: packages/web/README.md [hunk 1]
```diff
@@ -60,6 +62,11 @@ type Config = {
   debug: bool; // enable or disable useful console.log outputs
   proxyToWorker: bool; // Whether to proxy the calculations to a web worker. (Default true)
   model: 'small' | 'medium'; // The model to use. (Default "medium")
+  output: {
+    type: 'image/png' | 'image/jpeg' | 'image/webp'; // The output type. (Default "image/png")
```
Comment (mirko314) [inline packages/web/README.md:66]: Is this correct? It seems like that should 
```suggestion
    format: 'image/png' | 'image/jpeg' | 'image/webp'; // The output type. (Default "image/png")
```

File: packages/web/README.md [hunk 2]
```diff
@@ -60,6 +62,11 @@ type Config = {
...
+    quality: number; // The quality. (Default: 0.8)
+    content: 'image' | 'alpha';
```
Comment (mirko314) [inline packages/web/README.md:68]: Is this correct?

File: packages/web/tsconfig.json [hunk 3]
```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "lib": ["DOM", "ES2015"],
     "resolveJsonModule": true,
+    // "declarationMap": true,
```
Comment (mirko314) [inline packages/web/tsconfig.json:5]: Should this be added?

File: packages/node/README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Background Removal in NodeJs
+\*\*\*\*# Background Removal in NodeJs
```
Comment (mirko314) [inline packages/node/README.md:1]:
```suggestion
# Background Removal in NodeJs
```

**Review by mirko314 — CHANGES_REQUESTED** [review]
Awesome work! This will enable a lot more workflows!

The NodeJs example and Vite example worked great.
Some minor documentation issues arose:

File: packages/web/README.md (hunk 1 above)
Comment (DanielHauschildt) [reply packages/web/README.md]: Good point. There was another error

File: packages/web/README.md (hunk 2 above)
Comment (DanielHauschildt) [reply packages/web/README.md]: No, I fixed it. Thx

File: packages/web/tsconfig.json (hunk 3 above)
Comment (DanielHauschildt) [reply packages/web/tsconfig.json:5]:
```suggestion
```

File: packages/web/README.md [hunk 5]
```diff
@@ -166,7 +166,7 @@ let config: Config = {
 
 ## Who is it for?
 
-`@imgly/background-removal` is ideal for developers and projects that require efficient and cost-effective background removal directly in the browser. It caters to a wide range of use cases, including but not limited to:
+`@imgly/background-removal` is ideal for developers and projects that require efficient and cost-effective background removal directly in sthe browser. It caters to a wide range of use cases, including but not limited to:
```
Comment (mirko314) [inline packages/web/README.md:169]:
```suggestion
`@imgly/background-removal` is ideal for developers and projects that require efficient and cost-effective background removal directly in the browser. It caters to a wide range of use cases, including but not limited to:
```

**Review by mirko314 — CHANGES_REQUESTED** [review]
Found only one typo, rest looks good!

**Pull request merged**